cmd/webhook/admission/testdata/rg/rg-with-invalid-backend-path.json

```diff
@@ -0,0 +1,98 @@
+{
+  "request": {
+    "uid": "req-uid",
+    "name": "req1",
+    "operation": "create",
+    "kind": {
+      "group": "zalando",
+      "version": "v1",
+      "kind": "RouteGroup"
+    },
+    "namespace": "n1",
+    "object": {
+      "metadata": {
+        "name": "rg1",
+        "namespace": "n1"
+      },
+      "spec": {
+        "backends": [
+          {
+            "name": "valid-backend1",
+            "type": "shunt"
+          },
+          {
+            "name": "valid-backend3",
+            "type": "network",
+            "address": "http://example.com"
+          },
+          {
+            "name": "invalid-backend2",
+            "type": "network",
+            "address": "http://example.com/foo"
+          },
+          {
+            "name": "invalid-backend3",
+            "type": "network",
+            "address": "http://example.com/foo/bar"
+          },
+          {
+            "name": "invalid-backend4",
+            "type": "network",
+            "address": "http://example.com/foo/"
+          },
+          {
+            "name": "invalid-backend5",
+            "type": "network",
+            "address": "/foo"
+          },
+          {
+            "name": "valid-backend2",
+            "type": "network",
+            "address": "http://user:pass@example.com"
```

Comment:
Does "http://user@example.com" make sense as valid address? If it makes sense, should we add it as an example of valid backend as well?

Reply:
we already covered `http://user:pass@example.com` which implies that `http://user@example.com` is a valid one I don't think we need to add extra test cases there. It won't hurt but since it's already passes [`url.Parse`](https://go.dev/play/p/lflJeQc4nYx) and there is no explicit condition to make it fail then no need to cover.